src/cpp.cc

```diff
@@ -21,48 +21,44 @@
 
 }}} */
 
+#include "cpp.hh"
+
 #include <cmath>
+#include <list>
 #include <set>
+#include <string>
 #include <tuple>
 #include <utility>
 #include <vector>
-#include <string>
-#include <list>
 #include <boost/tokenizer.hpp>
 
-#include "cpp.hh"
-
+#include "ast.hh"
+#include "cyk.hh"
+#include "expr.hh"
+#include "expr/new.hh"
+#include "fn_def.hh"
+#include "grammar.hh"
+#include "options.hh"
+#include "outside/codegen.hh"
 #include "statement.hh"
 #include "statement/backtrace_decl.hh"
+#include "statement/fn_call.hh"
 #include "statement/hash_decl.hh"
 #include "statement/marker_decl.hh"
-#include "statement/fn_call.hh"
 #include "statement/while.hh"
-#include "expr.hh"
-#include "expr/new.hh"
 #include "type.hh"
 #include "type/backtrace.hh"
 #include "type/multi.hh"
-
-#include "fn_def.hh"
-
 #include "var_acc.hh"
-
-#include "ast.hh"
-#include "grammar.hh"
-
-#include "options.hh"
-#include "outside/codegen.hh"
-#include "cyk.hh"
 #include "version.hh"
 
-
 static std::string make_comments(const std::string &s, const std::string &c) {
   std::ostringstream o;
   boost::char_separator<char> nl("\n", "", boost::keep_empty_tokens);
   boost::tokenizer<boost::char_separator<char> > lines(s, nl);
   for (boost::tokenizer<boost::char_separator<char> >::iterator i =
-       lines.begin(); i != lines.end(); ++i) {
+           lines.begin();
+       i != lines.end(); ++i) {
```

Comment:
why this change?
If you care about code style - and I am a fan of this - we might want to split your PR into one that adds functionality and a second one only addressing code style changes